buildpacks/ruby/src/layers/bundle_install_create_path_layer.rs

```diff
@@ -0,0 +1,109 @@
+use crate::RubyBuildpackError;
+use libcnb::data::layer_content_metadata::LayerTypes;
+use std::path::Path;
+
+use crate::RubyBuildpack;
+use libcnb::build::BuildContext;
+use libcnb::layer::{ExistingLayerStrategy, Layer, LayerData, LayerResult, LayerResultBuilder};
+
+use libcnb::data::buildpack::StackId;
+use libcnb::layer_env::{LayerEnv, ModificationBehavior, Scope};
+use serde::{Deserialize, Serialize};
+
+/*
+
+# Set up path for `bundle install` dependencies
+
+Must run before `execute_bundle_install_layer` to create directory for dependencies.
+
+## Layer dir: Create directory for dependencies
+
+Dependencies installed via `bundle install` will be stored in this layer's directory.
+This is accomplished via configuring bundler via environment variables
+
+## Set environment variables
+
+- `BUNDLE_BIN=<layer-dir>/bin` - Install executables for all gems into specified path.
+- `BUNDLE_PATH=<layer-dir>` - Directs bundler to install gems to this path
+- `GEM_PATH=<layer-dir>` - Tells Ruby where gems are located. Should match BUNDLE_PATH.
+
+Other environment variables for bundler are configured by another layer that is decoupled
+from dependency storage on disk to miminimize the risk of having to clear dependencies
+to update an environment variable.
+
+*/
+pub struct BundleInstallCreatePathLayer {
```

Comment:
Similar to above, maybe `BundlerPathLayer` or something still gets the point across without being specific to `bundle install` and so long?